configmock.yml

```diff
@@ -37,8 +37,8 @@ postgres:
 # [BNB] Binance DEX: https://wallet.binance.org
 #       Binance Chain: https://explorer.binance.org
 binance:
-  api: http://localhost:3347/binance-api/v1
-  dex: http://localhost:3347/binance-dex
+  api: https://dex.binance.org/api
+  dex: https://localhost:3347/api/v1
```

Comment:
localhost:3347 is the mock API server, for **ALL** mock APIs, therefore mock API paths have to be prefixed with the platform, like this:  http://localhost:3347/binance-rpc  (or -dex).  Just using 'localhost:3347/api/v1' for a mock path is wrong.

Note: mock script contain there path in the 2nd line usually:
module.exports = {  path: '/binance-rpc/...

Note: When `start-platform-api-mock` is started, it displays diagnostic output on the console about mock API calls received.